docs/dev/intermediate/custom-config-syncing.md

```diff
@@ -30,31 +30,41 @@ Add an **Assembly Reference** to the following files:<br>
 
```

Comment:
Consider changing some `LogInfo` to `LogDebug` to prevent normal players from having their log filled up.